src/main/java/core/basesyntax/controller/ConsoleHandler.java

```diff
@@ -0,0 +1,42 @@
+package core.basesyntax.controller;
+
+import core.basesyntax.dao.BetDao;
+import core.basesyntax.dao.UserDao;
+import core.basesyntax.lib.Inject;
+import core.basesyntax.model.Bet;
+import core.basesyntax.model.User;
+import java.util.Scanner;
+
+public class ConsoleHandler {
+    @Inject
+    BetDao betDao;
+    @Inject
+    UserDao userDao;
+
+    public void handle() {
+        Scanner scanner = new Scanner(System.in);
+        while (true) {
+            String command = scanner.nextLine();
+            if (command.equalsIgnoreCase("q")) {
+                return;
+            }
+            Bet bet = null;
+            User user = null;
+            try {
+                String[] data = command.split(" ");
+                String name = data[0];
+                int age = Integer.parseInt(data[1]);
+                int value = Integer.parseInt(data[2]);
+                double risk = Double.parseDouble(data[3]);
+                user = new User(name, age);
+                bet = new Bet(value, risk);
```

Comment:
Try to avoid massive try-catch blocks. There is no need to put all of this code in the brackets. Let's try to extract the part of it that doesn't provoke an exception.

Reply:
for example? 
              String[] data = command.split(" ");
              user = new User(name, age);
              bet = new Bet(value, risk);
it doesen't provoke, however we initialize them in try and can use only in this block

Reply:
We can declare the variable without initializing, agree?
Then you initialize them if the input data is correct.
If not - you're letting the user know that they've made a mistake, 
ask them to try again or quit the service.

Reply:
okay, we declare the variable before try and we also need to initialize them in try block if don't wanna to have NumberFormatException.
Therefore our try block will decrase only by 2 lines. 
I think 2 lines in our try block is not a big deal, doesen't it?

Reply:
String[] data = command.split(" ");

This line only needs to be put into the try block

Reply:
int age = Integer.parseInt(data[1]);
int value = Integer.parseInt(data[2]);
double risk = Double.parseDouble(data[3]);
Parse method can provoke an exception.
It was in video from hw

Reply:
Parsing itself won't provoke any exception in case of having correct input data
That's just impossible in our task

Reply:
Congrats on approving! :)
Don't think I'm trying to make you mad, these are just suggestions for future ♡

Reply:
No, i thanks a lot for your suggestion) It's cool to dispute and learn from mistakes)